packages/stryker-webpack-transpiler/test/unit/compiler/ConfigLoaderSpec.ts

```diff
@@ -0,0 +1,51 @@
+import * as path from 'path';
+import { expect } from 'chai';
+import ConfigLoader from '../../../src/compiler/ConfigLoader';
+import { Configuration, Compiler, Plugin } from 'webpack';
+import { createStrykerWebpackConfig } from '../../helpers/producers';
+
+class FooPlugin implements Plugin { foo = true; apply(compiler: Compiler): void { } }
+class ProgressPlugin implements Plugin { apply(compiler: Compiler): void { } }
+class BarPlugin implements Plugin { bar = true; apply(compiler: Compiler): void { } }
+
+describe('ConfigLoader', () => {
+  let sut: ConfigLoader;
+  let requireStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    requireStub = sandbox.stub();
+    sut = new ConfigLoader(requireStub);
+  });
+
+  it('should load webpack config from given location', () => {
+    requireStub.returns('resolved');
+    const result = sut.load(createStrykerWebpackConfig({ configFile: 'webpack.foo.config.js' }));
+    expect(result).eq('resolved');
+    expect(requireStub).calledWith(path.resolve('webpack.foo.config.js'));
+  });
+
+  it('should remove "ProgressPlugin" if silent is `true`', () => {
+    // Arrange
```

Comment:
Why are there suddenly comments here?

Reply:
Because the tests  has 8 lines of code. I usually add them when there are more lines, just to keep it clean.